.claude/__tests__/settings.local.test.js

```
@@ -0,0 +1,398 @@
+      expect(process.env.CLAUDE_API_KEY).toBe('env-api-key');
+      expect(process.env.CLAUDE_DEBUG).toBe('true');
+      
+      // Restore original environment
+      process.env = originalEnv;
+    });
+
+    test('should handle missing environment variables gracefully', () => {
+      const originalEnv = process.env;
+      
+      // Remove environment variables
+      delete process.env.CLAUDE_API_KEY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)